near/src/index.tsx

```diff
@@ -25,22 +25,22 @@ async function initConnection() {
   // Needed to access wallet
   const wallet = new WalletConnection(near, null);
 
-  const api = await initNear(wallet.account(), { contractId: 'filecoin-bridge.testnet' });
+  await requestSignIn(wallet);
```

Comment:
this is new, we must request sign in before we start the app, so that this is the same as for eth